.github/workflows/e2e.yml

```
@@ -136,9 +137,10 @@ jobs:
           DATASOURCE: ${{ matrix.datasources }}
         working-directory: .
 
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         if: always()
         with:
-          name: playwright-report-typescript
+          name: playwright-report-typescript-${{ matrix.runs-on }}
```

actionlint (1.7.4)

property "runs-on" is not defined in object type {datasources: string; frameworks: string; node-version: number; os: string; python-version: number}

(expression)